src/Abstractions/OrchestrationOptions.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Enum describing the runtime status of the orchestration.
+/// </summary>
+public enum OrchestrationRuntimeStatus
```

Comment:
I think we will need to have type-forwarding from this type in the client package.